katas/content/complex_arithmetic/powers_of_i/Verification.qs

```diff
@@ -0,0 +1,34 @@
+namespace Kata.Verification {
+   
+   operation PowersOfI_Reference(n : Int) : Int{
+    // If n is divisible by 4
+       if n % 4 == 0 { return 1;
+       }
+       else { return -1;
+       }
+   }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        mutable count = 1;
+        mutable n=0;
+        mutable success = true;
+        mutable actual = 0;
+        mutable expected = 0;
+
+        repeat {   
+            set n = 2*count;
+            set expected = PowersOfI_Reference(n);
+            set actual  = Kata.PowersOfI(n);
+            if expected != actual {set success = false;}
+            set count += 1;   
+        }
+        until (count > 25) or (success == false);   
+        if success == true { Message("Correct!");
+        }
+        else { Message("Incorrect solution. Result of exponentiation doesn't match expected value");
+               Message($"Actual value: {actual}. Expected value: {expected}");
```

Comment:
You need to tell them the value of the argument for which the answer is inaccurate, otherwise the message is not very helpful

Reply:
(The same comment applies for the rest of the tasks)